### PR TITLE
[Add] Alert handling

### LINKED ISSUE #16: Alert System

### PR DESCRIPTION
**Pull request by gsc2001**
Fixes #16

### CODE REVIEW

File: utils/hooks.ts [hunk 1]
```diff
@@ -1,4 +1,17 @@
-import { TypedUseSelectorHook, useSelector } from 'react-redux';
+import { TypedUseSelectorHook, useSelector, useDispatch } from 'react-redux';
```
Comment (tichnas) [inline utils/hooks.ts:1]: remove useSelector from here

File: utils/themes.ts [hunk 2]
```diff
@@ -8,26 +8,30 @@ import {
     DefaultTheme as PaperDefaultTheme,
 } from 'react-native-paper';
 
-const BaseTheme = {
-    roundness: 5,
```
Comment (tichnas) [inline utils/themes.ts:12]: why removed roundness?

Comment (gsc2001) [reply utils/themes.ts:12]: there is a built in roundness, this was overide i think I have added by mistake

File: utils/hooks.ts (hunk 1 above)
Comment (gsc2001) [reply utils/hooks.ts:1]: its needed see below

**Review by tichnas — APPROVED** [review]

**Pull request merged**